composer.json

```diff
@@ -28,9 +28,10 @@
         "php": ">=7.2.0",
         "phpunit/phpunit": "~8.0|~9.0",
         "mockery/mockery": "~1.1",
-        "symfony/var-dumper": "~3.0|~4.0|~5.0",
```

Comment:
Just wondering what do use for debug info outputing?

Reply:
You mean when debugging my code? Depends; xdebug, log files, vanilla var\_dump, but indeed also symfony/var-dumper, but as a global dependency. And since it is (and, more importantly, should not) be used in any pushed code, I removed it as a dependency of this library.